src/notification-groups/llvm.md

```diff
@@ -1,10 +1,10 @@
-# LLVM ICE-breakers
+# LLVM Notification group
 
-**Github Label:** [ICEBreaker-LLVM]
+**Github Label:** [A-LLVM]
```

Comment:
What about the `ICEBreaker-Cleanup-Crew` label? Will it stay the same? If not we should edit the reference to it in the `cleanup-crew.md`.

Reply:
I guess we should just change it to Cleanup-Crew

Reply:
I've merged this with the idea that the labels can be fixed in a follow-up PR.